### PR TITLE
Add C2PA Premade Sample

### PR DESCRIPTION
**Pull request by honourfish**
[AB#8102](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/8102)

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: archivist_samples/c2pa/c2pa.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+        custom_attrs: "dict|None" = None,
+    ):
+        props = {
+            "operation": "Record",
+            "behaviour": "RecordEvidence",
+        }
+        attrs = {
+            "arc_display_type": "Publish",
+            "arc_description": description,
+            "document_version_authors": authors,
```
Comment (henry739) [inline archivist_samples/c2pa/c2pa.py:142]: Something to note here is that we have weak validation for this in the backend. There is a feature about tightening that up - but users could enter some dodgy values for this. I believe the documentation on the website tells them how it should be created though.

File: archivist_samples/c2pa/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# C2PA Sample
+
+The purpose of this sample is to demonstrate how one can record and trace the lifecycle of an asset-embedded manifest within RKVST thus providing a historical chain of events of wanted or unwanted changes and/or updates.
```
Comment (JAG-UK) [inline archivist_samples/c2pa/README.md:3]:
```suggestion
The purpose of this sample is to demonstrate how one can record and trace the lifecycle of an embedded image manifest within RKVST thus providing a historical chain of events of wanted or unwanted changes and/or updates.

This gives a more complete view of the provenance of media files than relying only on the most recent embedded manifest.
```

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: archivist_samples/c2pa/c2pa.py (hunk 1 above)
Comment (honourfish) [reply archivist_samples/c2pa/c2pa.py:142]: I think that is a superb idea :) It took me by surprise the format of authors. I will document the format.

File: archivist_samples/c2pa/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Python 3.7 and later versions are supported.
+
+* Install the [RKVST samples Python package](https://pypi.org/project/rkvst-samples/ "PyPi package page")
+
+* Get an authorization bearer token and store it in the file `credentials/.auth_token`. If you don't know how to do this, please refer to the [RKVST documentation](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/ "Getting an auth token"). Make sure that the `credentials` folder is suitably restricted by disallowing root and group access.
+
+
+## Running the sample
+
+First acquire an auth token and put it in the following file:
```
[the remark this answers is not in the document]
Comment (honourfish) [reply archivist_samples/c2pa/README.md:17]: Good spot! will change :)

File: archivist_samples/c2pa/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# C2PA Sample
+
+The purpose of this sample is to demonstrate how one can record and trace the lifecycle of an asset-embedded manifest within RKVST thus providing a historical chain of events of wanted or unwanted changes and/or updates.
+
+
```
[the remark this answers is not in the document]
Comment (honourfish) [reply archivist_samples/c2pa/README.md:5]: Excellent suggestion! will add.

File: archivist_samples/c2pa/c2pa.py (hunk 1 above)
Comment (honourfish) [reply archivist_samples/c2pa/c2pa.py:145]: have added the expected authors format to the function comment.

File: archivist_samples/c2pa/README.md (hunk 2 above)
Comment (honourfish) [reply archivist_samples/c2pa/README.md]: done :) good suggestion!

File: archivist_samples/c2pa/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# C2PA Sample
+
+The purpose of this sample is to demonstrate how one can record and trace the lifecycle of an embedded file manifest within RKVST thus providing a historical chain of events of wanted or unwanted changes and/or updates.
+
+Note: For clarity an simplicity this sample does not show how to create the C2PA manifest itself, but rather uses pre-prepared files generated by the standard c2patool.
```
Comment (JAG-UK) [inline archivist_samples/c2pa/README.md:5]:
```suggestion
Note: For clarity and simplicity this sample does not show how to create the C2PA manifest itself, but rather uses pre-prepared files generated by the standard c2patool.
```

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: archivist_samples/c2pa/README.md (hunk 5 above)
Comment (honourfish) [reply archivist_samples/c2pa/README.md]: good spot! done

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**